### ISSUE TITLE
ssl issues

### ISSUE THREAD

**Issue by javaknight** [angelo-v/wordpress-backup#3]
I am having problems using this in conjuction with jwilder/nginx-proxy  ..  

What I am attempting to do is a site restore from a bitnami wordpress install.

First I have tested that this works just fine out of the box with SSL using vanilla wordpress from Docker.

The issue arises when I attempt to restore my Wordpress from a Bitnami stack wordpress. 

The install appears fine, until I enable SSL on Nginx-proxy. Then, it goes into a redirect loop. 

If I re-copy the vanilla wp-config.php into the migrated image, I get redirected to  https://127.0.0.1/wordpress/wp-admin/.

I have tried tinkering with wp-config.php with no real luck - although I am unsure how to enable the PHP logs on the docker vanilla image.

**Comment by javaknight** [comment]
going to rebuild the blog from the vanilla image upwards. I'll close this ticket.

**Comment by javaknight** [comment]
After rebuilding the site by hand I discovered the source of trouble. I wanted to share my findings with others in case they too encounter the issue.

Im my particular case, I was attempting to restore a WooCommerce site from an external gz dump. To recap, the site loaded in just fine, but was having an infinite redirect loop, or white screen, when I turned on SSL via the directions provided (place the cert in the folder, etc)

I discovered a setting I had left checked in WooCommerce. The setting is under WooCommerce in the Checkout Tab (4th tab from the left). The setting is called Force HTTP when leaving the checkout. Turns out that setting was enabled.

This setting was causing a redirect loop. Nginx-proxy out of the box wants to redirect everything (all pages) to HTTPS, and WooCommerce wants to redirect all non checkout pages to HTTP. So you have a loop that never ends.
